types/node/test/http.ts

```diff
@@ -112,6 +113,9 @@ import * as net from 'net';
 
     // event
     req.on('data', () => { });
+
+    // path
+    assert.equal(req.path.charAt(0), '/');
```

Comment:
type tests don't need to assert,here should be `const path: string = req.path;`

Reply:
I misunderstood how the test works. I've committed the fix (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33081/commits/4f17556377a38429be3c3fe456eb8ea9005079b4). Thank you.

Reply:
you can look here for more info https://github.com/Microsoft/dtslint#write-tests

Reply:
I got it. Thank you for the link!